lib/consts.dart

```diff
@@ -22,6 +22,36 @@ const kNetworkEndian = Endian.little;
 const kDbName = "forest_park_reports";
 const kImageDirectory = "images";
 
+/// An assortment of calculated values regarding the panel  
+class PanelValues {
```

Comment:
I'm not sure this is the best fit for consts.dart (specifically having functions in consts.dart). Maybe this could go in util/panel_values.dart, and the actual constants could go here (kPanelOpenFraction, kPanelCollapsedHeight, kPanelSnapPoint).